builtin/logical/database/path_rotate_role_credentials.go

```diff
@@ -69,6 +60,37 @@ func (b *databaseBackend) pathRotateRoleCredentialsUpdate() framework.OperationF
 	}
 }
 
+func (b *databaseBackend) populateRotationQueue(ctx context.Context, s logical.Storage) {
+	log := b.Logger()
+
+	b.Lock()
+	defer b.Unlock()
+
+	roles, err := s.List(ctx, "role/")
+	if err != nil {
+		log.Warn("unable to list role for enqueueing", "error", err)
+	}
+
+	for _, roleName := range roles {
+		role, err := b.Role(ctx, s, roleName)
+		if err != nil {
+			log.Warn("unable to read role", "error", err, "role", roleName)
+			continue
+		}
+		if role == nil {
+			continue
+		}
+
+		if err := b.credRotationQueue.PushItem(&queue.Item{
+			Key:      roleName,
+			Value:    role,
+			Priority: role.StaticAccount.LastVaultRotation.Add(role.StaticAccount.RotationFrequency).Unix(),
```

Comment:
I imagine this panics on non-static accounts. Adding the `nil` check above should skip this part, yeah?

Reply:
👍